esmvalcore/cmor/_fixes/native6/ipsl_cm6.py

```diff
@@ -0,0 +1,119 @@
+"""Fixes for IPSLCM6 TS output format."""
+import logging
+import os
+import time
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+logger = logging.getLogger(__name__)
+
+# The key used in mappings.yml file for providing the
+# variable name (in NetCDF file) that match the CMOR variable name
+KEY_FOR_VARNAME = "ipsl_varname"
+
+
+class AllVars(Fix):
+    """Fixes for all IPSLCM variables."""
+    def fix_file(self, filepath, output_dir):
+        """Select IPSLCM variable in filepath, by calling CDO, if relevant.
+
+        This is done only if input file is a multi-variable one. This is
+        diagnosed by searching in the input filepathame for the mapping
+        value for key 'group'.
+
+        In such cases, it is worth to use an external tool for
+        filtering, at least until Iris loads fast (which is not the case
+        up to, and including, V3.0.2)
+
+        However, we take care of ESMValTool policy re. dependencies licence
+        """
+        if "_" + self.var_mapping.get("group",
```

Comment:
I see several uses of `self.var_mapping` but I can not find the assigmment

Reply:
This PR anticipates on a merge of #1124, which assigns var_mapping through reading a mapping file (this means that IPSL-CM data cannot be processed right now, as this would raise a bug on this code line) 
I tried to explain that, and why, in the [introductory text for the PR](https://github.com/ESMValGroup/ESMValCore/pull/1153#issue-658181932). 
I tested the whole on a local branch which included #1124. I can push such a branch to the repository if useful.